kube-runtime/src/controller/mod.rs

```diff
@@ -268,15 +268,16 @@ where
         move |s| {
             Runner::new(scheduler(s), move |request| {
                 let request = request.clone();
+                let reconciler_span = info_span!("reconciling object", "object.ref" = %request.obj_ref, object.reason = tracing::field::Empty);
```

Comment:
I think this looks sensible.
The span now wraps one extra call, but that should be an instant one.

Reply:
We haven't actually entered the `span` here, so it does nothing until we call `record` on it - so I could actually avoid `record` entirely. Let me fix it.